### ISSUE TITLE
how to generate the filtered outputs in each step of training

### ISSUE THREAD

**Issue by Sissuire** [ZHKKKe/Harmonizer#3]
Hi. Good work and thanks for your sharing. 
I go through the paper and the code, but I am still not familiar with how to generate the intermediate images (i.e., the $I_k$ ) for the training thus calculate the loss function. Did I just miss it? Or could you plz give more details.

**Comment by ZHKKKe** [comment]
Hi, thanks for your attention.

You may use the following function to create the intermediate images for training: 
https://github.com/ZHKKKe/Harmonizer/blob/62222ace055449e088c14b954a9f0e26887f3643/src/model/module.py#L70
(You can randomly generate the input `arguments`).

**Comment by Sissuire** [comment]
@ZHKKKe Thanks for your kindly reply. My confusion has been cleared.
